.github/workflows/verify-links.yml

```
@@ -0,0 +1,69 @@
+name: Verify links
+
+on:
+  push:
+    branches:
+      - main
+      - workflow/verify-links # TODO REMOVE
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "08 08 * * 1"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write # required for peter-evans/create-issue-from-file
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Restore lychee cache
+        uses: actions/cache@v4
+        with:
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          fail: false
+          # Exclude all private and local addresses in the check
+          # Also exclude domains that actively block GitHub to send requests
+          # Any Cockpit URLs it finds that have variables will be ignored too
+          args: |
+            --max-concurrency 1
+            --retry-wait-time 60
+            --base .
+            --skip-missing
+            --exclude-all-private
+            --exclude '^https://linux.die.net'
+            --exclude 'file:///'
+            --exclude 'domain.tld'
+            --exclude '^.*\{\{'
+            --exclude 'https://bodhi.fedoraproject.org/updates/cockpit-*'
+            --cache
+            --cache-exclude-status 400..=599
+            --max-cache-age 1d
+            -v
+            .
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Find the last open report issue
+        if: steps.lychee.outputs.exit_code != 0
+        id: last-issue
+        uses: micalevisk/last-issue-action@v2
```

This one seems to be dead for 2 years without releases and I am slightly concerned about supply chain attacks here.

It essentially does [one query](https://github.com/micalevisk/last-issue-action/blob/main/src/core.ts#L23) which seems to be trivially rewritten in JavaScript in the workflow like we do for `dependabot` :
```
            try {
              await github.rest.issues.removeLabel({
                owner: context.repo.owner,
                repo: context.repo.repo,
                issue_number: context.issue.number,
                name: 'node_modules'
              });
            } catch (e) {
              if (e.name == 'HttpError' && e.status == 404) {
                /* expected: 404 if label is unset */
              } else {
                throw e;
              }
            }
```

But then `github.rest.issues` using [actions/github-script](https://github.com/actions/github-script?tab=readme-ov-file#comment-on-an-issue)

Getting the issue might be as simple as `octokit.rest.issues.listForRepo({ labels: ["link-checker"] })` and we always create an urls-check issue with a fixed label.

---

We could use the hash for taking the release and update it with Renovate when there are new versions (if ever). Would at least ensure that the release itself is the one we expect.
```suggestion
        uses: micalevisk/last-issue-action@044e1cb7e9a4dde20e22969cb67818bfca0797be # v2
```

But yeah might be better to use something like that if it works